pythran/backend.py

```diff
@@ -1137,8 +1137,11 @@ def visit_IfExp(self, node):
         test = self.visit(node.test)
         body = self.visit(node.body)
         orelse = self.visit(node.orelse)
-        return ("(pythonic::__builtin__::functor::bool_{{}}"
-                "({0}) ? {1} : {2})".format(test, body, orelse))
+
+        return ("pythonic::__builtin__::pythran::ifexp((bool){0}, {1}, {2})"
```

Comment:
Looking at the implementation of ``__builtin__::bool_``, I would say it doesn't but may be I am wrong.